app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class PostsController < ApplicationController
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = current_user.posts.build(post_params)
+    if @post.save
+      flash[:success] = 'Your post has been created'
+      redirect_to posts_index_path
+    else
+      render 'new'
+    end
+  end
+
+  def index
```

Comment:
- [ ] Index displays posts in a timeline (recent to older). We can achieve this by using [default scope](https://apidock.com/rails/ActiveRecord/Base/default_scope/class) in `post.rb`. An example could be [this](https://stackoverflow.com/questions/3393687/default-sort-order-for-a-rails-model).